src/vcBPA.cpp

```diff
@@ -99,7 +98,7 @@ struct vcBPAGrid
 
 struct vcBPAManifold
 {
-  udChunkedArray<vcBPAGrid> grids;
+  udChunkedArray<vcBPAGrid*> grids;
```

Comment:
Yes, this doesn't make sense out of context either, the idea is that grids wont be stored like this at the end of the changes, they'll be referenced only by convertItemData structs

Reply:
If you were to insist on having all changes in these PRs be indicative of the final code and also make sense btw it would look exactly the same as the "core changes" PR you closed